### ISSUE TITLE
Add explanation of FSLeyes installation/integration for Mac/Linux users

### ISSUE THREAD

**Issue by ahill187** [spinalcordtoolbox/spinalcordtoolbox#3223]
Currently, in the docs, under the `Installation` section, there is a description of how to install SCT with FSLeyes: https://spinalcordtoolbox.com/en/stable/user_section/installation.html#install-on-windows-10-with-wsl (under 4. OPTIONAL: Install FSLeyes). However, there is no description for Mac or Linux users.

## Suggestion

I would move the FSLeyes instructions into their own separate section, and then describe how to install on Mac, Windows, and Linux. Perhaps this could go after `Matlab Integration with Mac`?

**Comment by jcohenadad** [comment]
thank you for the excellent suggestion @ahill187 !

i would personally put it right after https://spinalcordtoolbox.com/en/stable/user_section/installation.html#install-with-pip-experimental, and remove https://spinalcordtoolbox.com/en/stable/user_section/installation.html#hard-core-installation-less-sct-usage (eg. move it as a new issue, or in the wiki), because i'm pretty sure nobody on this planet has ever used that installation mode (or maybe one person: @zougloub 😉 )

**Comment by joshuacwnewton** [comment]
This issue was addressed by #3228.

----

However, before I close this issue, I want to mention that I don't think that we should keep these instructions. 

The reason is because of issue https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3262. In that issue, we came to the conclusion that [FSLeyes should be separate from `venv_sct`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3262#issuecomment-793149993). And, to that end, I wrote the following on [this page](https://spinalcordtoolbox.com/user_section/fsleyes.html):

> Previously, SCT provided instructions on how to install FSLeyes into the SCT environment. However, we now request that you install FSLeyes separately and manage the installation on your own. You can find installation instructions for FSLeyes at this link.

Yet, after #3228, we now have this ["Installing FSLeyes" page](https://spinalcordtoolbox.com/user_section/installation/fsleyes.html) that recommends installing FSLeyes into the SCT environment, which is counter to the conclusion we came to.

So, I think we should remove this "Installing FSLeyes" page, leave the installation of FSLeyes to users.
